src/main/scala/io/qbeast/spark/utils/QbeastUtils.scala

```diff
@@ -72,10 +59,77 @@ object QbeastUtils extends Logging {
       .as[String]
       .collect()
 
-    log.info(s"Histogram for column $columnName: $histogram")
-    histogram
-      .map(string => s"'$string'")
-      .mkString("[", ",", "]")
+    log.info(s"String Quantiles for column $columnName: $quantiles")
+    quantiles.map(string => s"'$string'")
+
+  }
+
+  private def computeQuantilesForNumericColumn(
+      df: DataFrame,
+      columnName: String,
+      numberOfQuantiles: Int,
+      relativeError: Double): Array[Double] = {
+    val probabilities = (1 to numberOfQuantiles).map(_ / numberOfQuantiles.toDouble).toArray
```

Comment:
If `numberOfQuantiles` is 2, the probabilities will be `Array(0.5, 1.0).`
If `numberOfQuantiles` is 10, the probabilities will be `Array(0.1, 0.2, ..., 1.0).`
In other words, we are always leaving the minimum value out, but always including the maximum value.

Reply:
Should we remove as well the maximum value?

Reply:
Discussed:
1. We include 0.0 and 1.0
2. Should be > 1
3. Do not remove duplicates on String resolution